cypress/integration/visitor/loginPage/signIn.spec.js

```diff
@@ -1,19 +1,38 @@
 describe('Sign In', () => {
   beforeEach(() => {
-    cy.fixture('users').then(users => {
+    cy.fixture('users').then((users) => {
       this.users = users;
     });
   });
 
-  it('Visitor singns in with valid credentials', () => {
+  afterEach(() => {
+    cy.get('body').then(($body) => {
+      if ($body.find('[data-cy=dropdown-toggler]').length > 0) {
+        cy.get('[data-cy=dropdown-toggler]').click();
+        cy.get('[data-cy=sign-out]').click();
+      }
+    });
+  });
+
+  it('Visitor signs in with valid credentials', () => {
     const validCredentials = this.users.validUser;
 
     cy.login(validCredentials);
 
     cy.get('[data-cy=dropdown-toggler]').should('contain', validCredentials.email);
   });
 
-  it('Visitor singns in with invalid credentials', () => {
+  it('Visitor should be redirected to home page if authorized', () => {
```

Comment:
This case for an authorized user, not for visitor

Reply:
Please pay attention to name of the test, it should sound like scenario: `User visits auth page